libs/positions/src/lib/positions-table.tsx

```diff
@@ -396,20 +390,14 @@ export const PositionsTable = forwardRef<AgGridReact, Props>(
               valueGetter: ({ data }: VegaValueGetterParams<Position>) => {
                 return !data
                   ? undefined
-                  : toBigNum(
-                      data.unrealisedPNL,
-                      data.marketDecimalPlaces
-                    ).toNumber();
+                  : toBigNum(data.unrealisedPNL, data.decimals).toNumber();
               },
               valueFormatter: ({
                 data,
               }: VegaValueFormatterParams<Position, 'unrealisedPNL'>) =>
                 !data
                   ? ''
-                  : addDecimalsFormatNumber(
-                      data.unrealisedPNL,
-                      data.marketDecimalPlaces
-                    ),
+                  : addDecimalsFormatNumber(data.unrealisedPNL, data.decimals),
```

Comment:
```suggestion
                  : addDecimalsFormatNumber(data.unrealisedPNL, data.decimals, data.marketDecimalPlaces),
```

Reply:
Not going to add this for now in favor of just fixing the issue lets consider it as a separate ticket